### ISSUE TITLE
Bug: Mattermost Board - Team Not Found

### ISSUE THREAD

**Issue by amyblais** [mattermost/focalboard#1804]
Original report by @tschlue: https://github.com/mattermost/mattermost-server/issues/18986

#### Summary
If I create a Board or try to enter one I always geht the error site

>Team Not Found
>The team you're requesting is private or doesn't exist. Please contact your Administrator for an
> invitation.

#### Steps to reproduce
* Goto an channel
* Push the board button once
* Push the button '+Board hinzufügen' (+Add board?)
* Choose 'Leeres board' (Empty board)
-> Error screen

#### Expected behavior
The board should load without error

Maybe it have something to do that I tried to run focalboard on the same server.
Is there a way to reset mattermost? I just updated to Version 6.0.2 with the hope it fixes it.
I have the version 0.9.4 of the Mattermost Boards plugin installed.

**Comment by tschlue** [comment]
Here the log during I reproduce the error:

`        {"timestamp":"2021-11-12 07:42:42.057 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/users/status/ids","request_id":"6jhf4674hib65jsck7x78wcbnh","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:42.614 -05:00","level":"debug","msg":"Websocket request","caller":"wsapi/websocket_handler.go:25","action":"user_update_active_status"}
{"timestamp":"2021-11-12 07:42:44.125 -05:00","level":"debug","msg":"Websocket request","caller":"wsapi/websocket_handler.go:25","action":"user_update_active_status"}
{"timestamp":"2021-11-12 07:42:44.193 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/users/status/ids","request_id":"8p5op1qj1pyyi8nwmfk3na1qya","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:45.449 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/users/status/ids","request_id":"bix1tb7w33d4bq1mqw4x5rdi1e","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:45.526 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/users/status/ids","request_id":"j3ccrwhrfpnetyy5h853zyupqa","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:46.312 -05:00","level":"debug","msg":"Websocket request","caller":"wsapi/websocket_handler.go:25","action":"user_update_active_status"}
{"timestamp":"2021-11-12 07:42:46.366 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/users/status/ids","request_id":"3g84ebeotjd73kmbifdpe3m9xr","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:50.881 -05:00","level":"debug","msg":"Websocket request","caller":"wsapi/websocket_handler.go:25","action":"user_update_active_status"}
{"timestamp":"2021-11-12 07:42:51.635 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/users/status/ids","request_id":"657athte9if57niiet1byzii1w","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:51.648 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/plugins/webapp","request_id":"oi5onghry7gnbywn3r6hyxiago","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:51.678 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/users/me/teams/h9s8g1jy17bk7qbwaqtwijqdco/channels","request_id":"gzyftns4sjnk5j35grwpbeak7c","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:51.681 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/users/me/teams/unread","request_id":"jrotdg1sepfbzq94hp18parpcy","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:51.681 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/channels/f556spzahjdqfxnb1bawuqob8w/posts","request_id":"u1z7o8xnj3dj3x8nnfx897hkke","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:51.682 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/users/me/teams/h9s8g1jy17bk7qbwaqtwijqdco/channels/members","request_id":"y15dnjryjir9xpeeim434j4djc","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:51.687 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/users/ids","request_id":"ubidpmafhtyxmjhxaw7mmic1wc","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:52.964 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/users/status/ids","request_id":"au8j3cyapf899efcqwzhun5bac","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:53.860 -05:00","level":"debug","msg":"Websocket request","caller":"wsapi/websocket_handler.go:25","action":"user_update_active_status"}
{"timestamp":"2021-11-12 07:42:54.981 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/users/status/ids","request_id":"3ky4wthy3pfc8jjgt57ne8mbmw","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.228 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/channels/members/me/view","request_id":"yqprsfynipfpde78jcn5rn3amr","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.286 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/users/me/teams/h9s8g1jy17bk7qbwaqtwijqdco/channels","request_id":"qgcxut6sjtygiqai3171hp4cmo","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.327 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/channels/members/me/view","request_id":"yhgxfs958fg1brf5hk4bh5ajor","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.328 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/users/me/teams/h9s8g1jy17bk7qbwaqtwijqdco/channels/members","request_id":"niifaogr8j8zzy8s1y1otgsk7a","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.349 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/channels/members/me/view","request_id":"3kunawhpo7yct8ys5kx1bkn4re","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.421 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/boards/workspace/1w6qfjjjff8fdd8yjxndwse5ye","request_id":"g9qczpawafri7dbu6oqcbjm1ka","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.421 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/plugins/webapp","request_id":"9yg1g64nnjbqzj6w4x8bkyf7no","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.422 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/config/client","request_id":"wnjkqxxbhifutrgtxgrgyzqkch","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.460 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/plugins/webapp","request_id":"nyz5osufo7rkzcye57fiyc99ue","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.461 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/config/client","request_id":"rykg7rxoef83prfbqycrjyxm3a","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.492 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/plugins/webapp","request_id":"y99zq5g4ntne8g3kz9r6fqigbw","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.493 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/config/client","request_id":"fgje3t4m4fb63mweahowbanwwr","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.625 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/channels/members/me/view","request_id":"t6yucp3ws3ny3m9tidmdidy5hw","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.952 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/config/client","request_id":"ptacj7uppigbxd47gf4ox73e8a","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.957 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/plugins/webapp","request_id":"s5dt9h184f8odrwxc9rxkhaqua","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.992 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/plugins/webapp","request_id":"a8b9uubqxpgq3yg7zik6nnhrry","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:55.993 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/config/client","request_id":"5wadyqw8kinzbk6e48juxgkr1y","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.022 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/config/client","request_id":"j7mxrakukpdzpbzbw1escr1yma","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.023 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/plugins/webapp","request_id":"p8ksqnbop38kpjo1p9krbehity","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.053 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/config/client","request_id":"sk6wqgyr4bgiznbmeznfrbfgny","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.054 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/plugins/webapp","request_id":"cn5fpwbjtjrh7bpnbszixf4eeh","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.225 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/config/client","request_id":"qmaw5btik3dduysppsna3y8pme","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.233 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/license/client","request_id":"1h169kpy5tf7fj33ea95i1b5or","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.368 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/users/me","request_id":"81qgrt9gu7y7xgh4q5x5a7okzc","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.371 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/users/me/preferences","request_id":"4xsdq7g3ejdtjc3bh6zba5t4qw","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.373 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/users/me/teams","request_id":"y9rtjm8j3jdh8bjhcstz1ewioy","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.373 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/users/me/teams/members","request_id":"a5bjrufco3g9tefsuo3womhihr","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.405 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/roles/names","request_id":"uipsfpb9apbydk71g6qijncbee","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.412 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/roles/names","request_id":"x6epnmj587r8xfjeffciybcgdy","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.442 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/users/me/teams/unread","request_id":"w97affexgj873cprfmhna9eezr","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.444 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/plugins/webapp","request_id":"4x1nhzbecjfz5q8nm17yx46mow","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.444 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/emoji/name/plus","request_id":"654iuatnjb8j3fxe6mudsemf5e","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.445 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/emoji/name/crazy","request_id":"d8odok66ofbumkmdi3u3m4swih","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.464 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/emoji/name/quadra","request_id":"x58fms3yktd75b5xp98auxpi7w","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.466 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/emoji/name/excellent","request_id":"3tqnftjyu7na7kgcj4d654b3na","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:56.535 -05:00","level":"info","msg":"reminder preference is empty. Defaulting to \"on\"","caller":"app/plugin_api.go:895","plugin_id":"com.mattermost.plugin-todo"}
{"timestamp":"2021-11-12 07:42:57.030 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/","request_id":"9dm7i18j4ty4irmugmc4b78jhr","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:57.068 -05:00","level":"info","msg":"debug [2021-11-12 07:42:57.068 -05:00] attachSession caller=\"mlog/mlog.go:213\" single_user=false\n","caller":"io/io.go:425","plugin_id":"focalboard","source":"plugin_stdout"}
{"timestamp":"2021-11-12 07:42:57.090 -05:00","level":"debug","msg":"inactive connection found for webconn, reusing","caller":"app/plugin_api.go:892","plugin_id":"focalboard","":"webConnID","":"","":"userID","":"c7up9ghhpjydzbaw7x1ebwo4zc"}
{"timestamp":"2021-11-12 07:42:57.121 -05:00","level":"debug","msg":"Command: SUBSCRIBE_WORKSPACE","caller":"app/plugin_api.go:892","plugin_id":"focalboard","":"webConnID","":"","":"userID","":"c7up9ghhpjydzbaw7x1ebwo4zc","":"workspaceID","":"1w6qfjjjff8fdd8yjxndwse5ye"}
{"timestamp":"2021-11-12 07:42:57.126 -05:00","level":"info","msg":"debug [2021-11-12 07:42:57.126 -05:00] attachSession caller=\"mlog/mlog.go:213\" single_user=false\n","caller":"io/io.go:425","plugin_id":"focalboard","source":"plugin_stdout"}
{"timestamp":"2021-11-12 07:42:57.128 -05:00","level":"info","msg":"debug [2021-11-12 07:42:57.127 -05:00] attachSession caller=\"mlog/mlog.go:213\" single_user=false\n","caller":"io/io.go:425","plugin_id":"focalboard","source":"plugin_stdout"}
{"timestamp":"2021-11-12 07:42:57.128 -05:00","level":"info","msg":"debug [2021-11-12 07:42:57.128 -05:00] attachSession caller=\"mlog/mlog.go:213\" single_user=false\n","caller":"io/io.go:425","plugin_id":"focalboard","source":"plugin_stdout"}
{"timestamp":"2021-11-12 07:42:57.129 -05:00","level":"info","msg":"debug [2021-11-12 07:42:57.129 -05:00] attachSession caller=\"mlog/mlog.go:213\" single_user=false\n","caller":"io/io.go:425","plugin_id":"focalboard","source":"plugin_stdout"}
{"timestamp":"2021-11-12 07:42:57.141 -05:00","level":"info","msg":"debug [2021-11-12 07:42:57.140 -05:00] GetBlocks caller=\"mlog/mlog.go:213\" parentID= blockType= block_count=12\n","caller":"io/io.go:425","plugin_id":"focalboard","source":"plugin_stdout"}
{"timestamp":"2021-11-12 07:42:57.175 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/","request_id":"qj9k7em9ep8ctdpih5fr5nm6uw","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:57.204 -05:00","level":"info","msg":"debug [2021-11-12 07:42:57.204 -05:00] attachSession caller=\"mlog/mlog.go:213\" single_user=false\n","caller":"io/io.go:425","plugin_id":"focalboard","source":"plugin_stdout"}
{"timestamp":"2021-11-12 07:42:57.208 -05:00","level":"info","msg":"debug [2021-11-12 07:42:57.207 -05:00] GetBlocks caller=\"mlog/mlog.go:213\" parentID= blockType=board block_count=5\n","caller":"io/io.go:425","plugin_id":"focalboard","source":"plugin_stdout"}
{"timestamp":"2021-11-12 07:42:57.660 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/users/status/ids","request_id":"h4srb47iwfg7me35gbi7muc1cr","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:59.299 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/channels/members/me/view","request_id":"9zoy1ugkytyxfxpkmuyjxd34dc","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:59.335 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/config/client","request_id":"ittuapezjtynimoguwbz439qyo","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:59.354 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/plugins/webapp","request_id":"96dbgfb3ajnp9pq5kgszn6zzzw","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:59.367 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/config/client","request_id":"zoxd5pyb43gjp89erz9z9zhy9e","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:59.387 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/plugins/webapp","request_id":"4e7jqwm7sfycmxqrufpezk1ufh","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:59.394 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/config/client","request_id":"1jrh8zr45fyk8mu7ywrgtwtp1a","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:59.428 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/plugins/webapp","request_id":"q65zdfi9ztb13yparefrpww3zr","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:59.434 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/config/client","request_id":"keg4x6x4xfbxbjkzh1ctt157xw","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:59.449 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/users/status/ids","request_id":"tu7hqsek93bp3pe87uocfqu53w","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:59.454 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/plugins/webapp","request_id":"p7g5o44eytbaumwegunq8kutra","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:59.495 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/channels/members/me/view","request_id":"9abqjgnr6pnpmrjoh936pk3uny","host":"de1ksedev051","scheme":"http","status_code":"200"}
{"timestamp":"2021-11-12 07:42:59.692 -05:00","level":"debug","msg":"Unable to find the existing team.","caller":"web/context.go:101","path":"/mattermost/api/v4/teams/name/mattermost","request_id":"r5iefawdhirf7y4rtn9ocn8iih","ip_addr":"10.181.16.255","user_id":"c7up9ghhpjydzbaw7x1ebwo4zc","method":"GET","err_where":"GetTeamByName","http_code":404,"err_details":"resource: Team id: name=mattermost"}
{"timestamp":"2021-11-12 07:42:59.692 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/teams/name/mattermost","request_id":"r5iefawdhirf7y4rtn9ocn8iih","host":"de1ksedev051","scheme":"http","status_code":"404"}
{"timestamp":"2021-11-12 07:42:59.731 -05:00","level":"debug","msg":"websocket.read: client side closed socket","caller":"app/web_conn.go:783","user_id":"c7up9ghhpjydzbaw7x1ebwo4zc"}
{"timestamp":"2021-11-12 07:42:59.731 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"GET","url":"/mattermost/api/v4/websocket","request_id":"mbigxqc3ppnc5g6po9n5w5k7rr","host":"de1ksedev051","scheme":"http"}
{"timestamp":"2021-11-12 07:43:01.347 -05:00","level":"debug","msg":"Received HTTP request","caller":"web/handlers.go:107","method":"POST","url":"/mattermost/api/v4/users/status/ids","request_id":"rmn38b5ikiniuxm71thgaapabh","host":"de1ksedev051","scheme":"http","status_code":"200"}
`

![grafik](https://user-images.githubusercontent.com/14072748/141470594-027458a1-e0a6-4835-97cb-18fcc72bd568.png)

**Comment by sbishel** [comment]
@tschlue - We have made quite a few fixes to routing since v0.9.4. Are you able to test this with 0.10.0 or 0.11.0?

**Comment by tschlue** [comment]
Hi I just tested it and it's still the same behavior .

**Comment by sbishel** [comment]
@tschlue Thanks for helping me troubleshoot. Could you provide a video of this? I would like to see how the Url is being set.

**Comment by tschlue** [comment]
![TeamNotFound](https://user-images.githubusercontent.com/14072748/144428593-703a2c33-56d2-456e-aca8-d5a6ca747594.gif)

**Comment by gabrielctn** [comment]
Same issue here !

**Comment by gabrielctn** [comment]
I had the same issue (https://github.com/mattermost/desktop/issues/2011) and I found that it came from my Mattermost being configured as a subdirectory instead of a subdomain. By the way, it came with another issue on the desktop version, being that if someone send a message with a link to a website belonging to another subdirectory of the same domain to that of Mattermost e.g https://www.example.fr/anothersite, while Mattermost is configured as https://www.example.fr/mattermost, well when a user clicked on the link, the page would open inside the mattermost app in either the Playbook tab or the Boards tab, with an error, instead of opening the link outside of the app, on a browser.

It seems like you also have configured your Mattermost as a subdirectory. I was personally able to ask to people managing our DNS to add en entry in order to have a subdomain instead of a subdirectory, and the only thing I had to change afterward in the configuration wad the `siteurl` in the `config.json` file, it worked like a charm.

Yet, I don't know if you have the possibility to do the same, and it is certainly not solving the issue in the first place.
